.travis.yml

```diff
@@ -1,17 +1,37 @@
 sudo: required
 dist: trusty
 
+services:
+  - docker
+
 language: java
 jdk:
   - openjdk8
   - oraclejdk8
 
 git:
   submodules: false
-before_install: scripts/travis-initialize-submodules
+before_install:
+  - scripts/run-xvfb
+  - scripts/travis-initialize-submodules
 
 install: ./gradlew setupDecompWorkspace
-script: ./gradlew build
+script: DISPLAY=:99 ./gradlew test --stacktrace
```

Comment:
We still need to run the other tests (e.g. license header checks) as well. Is there a reason you have changed this to run `test` instead of `build`?

Reply:
That was unintentional - I'll change it back.